services-core/src/solution_submission.rs

```diff
@@ -203,30 +197,23 @@ impl<'a> StableXSolutionSubmitting for StableXSolutionSubmitter<'a> {
         claimed_objective_value: U256,
         gas_price_cap: f64,
     ) -> Result<(), SolutionSubmissionError> {
-        // If the gas price cap is not at least the fast gas price then submitting the solution
-        // is not feasible because it would take too long.
-        match self.gas_price_estimating.estimate().await {
-            Ok(gas_price) => {
-                if gas_price_cap < gas_price {
-                    return Err(SolutionSubmissionError::Benign(format!(
-                        "Solution does not generate enough fees for the transaction \
-                             to execute quickly enough: price cap {} < fast gas price {}",
-                        gas_price_cap, gas_price,
-                    )));
-                }
-            }
-            Err(err) => log::warn!("failed to estimate gas price: {:?}", err),
-        }
+        let target_confirm_time = Instant::now()
+            + BatchId::from(batch_index)
+                .solve_end_time()
+                .duration_since(SystemTime::now())
+                .unwrap_or_default();
```

Comment:
What is the default here on `duration_since`? Is this something we need to be concerned with?

Reply:
It's zero but maybe it's clearer to specifically use zero instead of default. Will change to that.